boot/makebootable.go

```diff
@@ -356,6 +359,16 @@ func makeBootable20RunMode(model *asserts.Model, rootdir string, bootWith *Boota
 		if err := bl.InstallBootConfig(bootWith.UnpackedGadgetDir, opts); err != nil {
 			return fmt.Errorf("cannot install managed bootloader assets: %v", err)
 		}
+		// determine the expected command line
+		cmdline, err := ComposeCandidateCommandLine(model)
+		if err != nil {
+			return fmt.Errorf("cannot compose the candidate command line: %v", err)
+		}
+		modeenv.CurrentKernelCommandLines = bootCommandLines{cmdline}
+		// update the modeenv on disk
+		if err := modeenv.WriteTo(InstallHostWritableDir); err != nil {
```

Comment:
shouldn't we simply move the this out of the if and remove the previous WriteTo ?  writing modeenv twice, once partially is a bit strange to me

Reply:
pushed